packages/jaeger-ui/src/constants/default-config.tsx

```diff
@@ -66,6 +66,7 @@ export default deepFreeze(
         gaID: null,
         trackErrors: true,
       },
+      standardDatetimeFormat: FALLBACK_STANDARD_DATETIME_FORMAT,
```

Comment:
Since this is used in only one place, I don't think it's the "standard" date time format.

Reply:
I called it like that, because it overrides the format used by the STANDARD_DATETIME_FORMAT constant. Today, it's only used in one place, but if other bits of the UI start using https://github.com/jaegertracing/jaeger-ui/blob/master/packages/jaeger-ui/src/utils/date.tsx#L64 they will also be affected. I'm fine with calling it "traceStartDateFormat" too :)

Reply:
@tiffon What do you suggest? Shall I remove the config option and add the fractional seconds to the format as shown below? Or should I keep the config and just rename the variable to remove the standard_ prefix?